homeassistant/components/localtuya

```diff
@@ -0,0 +1 @@
+../../../localtuya-homeassistant/custom_components/localtuya/
```

Comment:
This seems astray.

Reply:
Yeah, will remove that. Not sure how it ended up there...